Assets/Text/JavaScript/TeritorryGrid.js

```diff
@@ -0,0 +1,40 @@
+/*
+  Grid generator for Teritorry Wars or Multiply || Release or stuffs.
+  Written by Box.
+*/
+
+class TeritorryGrid
+{
+  constructor (width, height, boxSize, x, y)
+  {
+    this.w = width;
```

Comment:
Could you please add some default values?
e.g.
```
this.w = width || 4;
```

Reply:
Ok I added soe def value for w, h, b. x, y doesn't required.